google/cloud/storage/mocks/mock_async_connection.h

```diff
@@ -16,8 +16,13 @@
 #define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_MOCKS_MOCK_ASYNC_CONNECTION_H
 
 #include "google/cloud/storage/async_connection.h"
+#include "google/cloud/storage/object_metadata.h"
```

Comment:
The base class (in `storage/async_connection.h`) must include or forward declare all of these.  IWYU is (again) asking us to do useless work.

Reply:
I'm just following the warnings on the import. Do you want me to revert these changes?